dashboard/config/locales/en.yml

```diff
@@ -437,6 +438,11 @@ en:
       current: "Current Levels"
       unused: "Unused Levels"
   upsell:
+    applab:
+      title: 'App Lab'
+      body: "App Lab is a programming environment to make simple apps that live on the web. It's easy: simply drag and drop to design your app, write code for your app with JavaScript blocks or with text, then share your app with friends in seconds."
```

Comment:
When I see a multi-step like this I think it might be clearer to read with bulleted numbers.